lona/html/node.py

```diff
@@ -177,11 +177,8 @@ def ignore(self):
     def ignore(self, value):
         if value:
             self._attributes['data-lona-ignore'] = 'True'
-
```

Comment:
I want to keep this newline to keep styling consistent with the rest of the code